microsoft/PowerToys
# [FZ Editor] Layout hotkeys for quick swap UI

## Summary of the Pull Request

**What is this about:**

Create UI to select a value from 0 to 9 to bind the current layout to the shortcut.

**What is include in the PR:** 

Added ComboBox to select the available key in the range from 0 to 9 for the shortcut.
Button to the right of the ComboBox resets selection.

ComboBoxes contains only free values that weren't applied to other layouts.

In the custom grid layout where items don't fit the window height and require a scrollbar.

![image](https://user-images.githubusercontent.com/8949536/110917505-08d5e500-832b-11eb-8021-1b20112d190b.png)

![image](https://user-images.githubusercontent.com/8949536/110917553-17240100-832b-11eb-9a3a-1901c8a47365.png)

**How does someone test / validate:** 

* Apply values to different layouts, check if they are presented and saved in settings correctly.
* Check that already applied values are not presented in the other layouts comboboxes.

## Quality Checklist

- [x] **Linked issue:** #10025

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
